CITATION.cff

```diff
@@ -3,6 +3,11 @@ title: pywaterinfo
 message: 'If you use this software, please cite it as below.'
 type: software
 authors:
+  - given-names: Stijn
+    family-names: ' Van Hoey'
```

Comment:
I don't think he space in the beginning is needed

Reply:
thanks, adjusted